cmd/kubeapps-apis/plugins/resources/v1alpha1/server.go

```diff
@@ -254,6 +255,34 @@ func (s *Server) GetResources(r *v1alpha1.GetResourcesRequest, stream v1alpha1.R
 	return nil
 }
 
+// GetResources returns the resources for an installed package.
+func (s *Server) GetServiceAccountNames(ctx context.Context, r *v1alpha1.GetServiceAccountNamesRequest) (*v1alpha1.GetServiceAccountNamesResponse, error) {
+	namespace := r.GetContext().GetNamespace()
+	cluster := r.GetContext().GetCluster()
+	log.Infof("+resources GetServiceAccountNames (cluster: %q, namespace=%q)", cluster, namespace)
+
+	typedClient, _, err := s.clientGetter(ctx, cluster)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "unable to get the k8s client: '%v'", err)
+	}
+
+	saList, err := typedClient.CoreV1().ServiceAccounts(namespace).List(context.TODO(), metav1.ListOptions{})
```

Comment:
We should be using the `ctx` that was passed in above, rather than creating a new context here?